providers/os/connection/snapshot/blockdevices.go

```diff
@@ -19,12 +20,12 @@ type BlockDevices struct {
 
 type BlockDevice struct {
 	Name       string        `json:"name,omitempty"`
-	FsType     string        `json:"FsType,omitempty"`
+	FsType     string        `json:"fstype,omitempty"`
```

Comment:
Is that caused by the fact that we switch to specific fields?

Reply:
I am really not sure. This is also `fstype` if you run `lsblk -f --json`. Don't know why this was specified like that